docs/core_concepts/0_draft_and_deploy/index.md

```diff
@@ -0,0 +1,39 @@
+# Draft and Deploy
+
+Windmill lets you to iterate on scripts, flows and apps before deploying them.
+
+Each script, flow or app can go through 3 states: local edit, draft and deployed.
```

Comment:
It's not really a state since they are concurrent.

They are saved (sometimes concurrently) in 3 fashions, by your browser in local storage, by the backend as a draft, and deployed